docs/setup/gke.tf

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "null_resource" "previous" {}
+
+resource "time_sleep" "wait_120_seconds" {
+  depends_on = [null_resource.previous]
+
+  create_duration = "120s"
+}
+
+data "google_client_config" "default" {}
+
+provider "kubernetes" {
```

Comment:
Do you need this `provider "kubernetes"` or this `data "google_client_config"` parts?

Reply:
Taken from the asm sample tf! https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/a9a69edaed7dac3b700ad48309d7922c722609bb/examples/simple_zonal_with_asm/main.tf#L23

Reply:
Gotcha, interesting, didn't know that. That's indeed documented in the ASM module page: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/master/modules/asm#usage. Good to know!